File: Unidad_III/Felipe/Python/include/Auto.py
class Auto:
    #Cosntructor 
    def __init__(self, Marca ="Chevrolet", Modelo= 2019, Kilometraje = 14000, Color="Azul"):
        self.Marca = Marca
        self.Modelo = Modelo
        self.Kilometraje = Kilometraje
        self.Color = Color
        
    def mostrar (self):
        print(self.Marca)
        print(self.Modelo)
        print(self.Kilometraje)
        print(self.Color)
    
    def pintar (self, Ncolor):
        self.Color = Ncolor
        
    def correr (self, recorrido):
        self.Kilometraje = self.Kilometraje + recorrido
        

File: Unidad_III/Felipe/C++/include/Auto.cpp
#include <Auto.hpp>

using namespace std;
//Costructores
Auto::Auto(void)
{
    marca = "KIA";
    modelo = "Seratto";
    year = 2017;
    kilometros= 30000;
    color = "Azul";
}

Auto:: Auto (string Cmarca, string Cmodelo, int Cyear, int Ckilometros, string CColor)
{
    marca = Cmarca;
    modelo = Cmodelo;
    year = Cyear;
    kilometros =Ckilometros;
    color = CColor;
}

//Metodos

void Auto::pintar(string a)
{
    color = a;
    cout<<"El color es"<<Auto::color<<endl;
    
}

void Auto::correr(int b)
{
    kilometros =kilometros+ b;
    cout<<"El kilometraje es"<<Auto::kilometros<<endl;
    
}

void Auto::modificar(string c)
{
    modelo= c;
    cout<<"El Modelo es"<<Auto::modelo<<endl;
    
}

void Auto::checar(void)
{
    cout<<"Marca"<<Auto::marca<<endl;
    cout<<"Año"<<Auto::year<<endl;
    cout<<"Color"<<Auto::color<<endl;
    cout<<"Kilometraje"<<Auto::kilometros<<endl;
    cout<<"Modelo"<<Auto::modelo<<endl;
    
}